src/main/java/de/retest/recheck/RecheckOptions.java

```diff
@@ -257,7 +257,7 @@ RecheckOptionsBuilder setIgnore( final Filter filter ) {
 		 * {@link #setIgnore(String)}.
 		 *
 		 * @param filtername
-		 *            The filter to add to the ignore.
+		 *            The filter to add to the ignore. The String needs the .filter-extension.
```

Comment:
Same as above I guess? Also shouldn't it be `filterName` as anywhere else (not your fault)?